repo/packages/C/confluent-replicator/3/config.json

```diff
@@ -0,0 +1,62 @@
+{

```

Comment:
Trying to keep the sub-services aligned, even though this is the first release of that service.